pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -55,7 +55,7 @@ export class JournalOutput {
         }
 
         // only consider enter button for keyboard events
-        if (ev.type === 'keypress' && ev.key !== "Enter")
+        if (ev.type === 'KeyDown' && ev.key !== "Enter")
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)